zhaquirks/tuya/ts0601_trv_sas.py

```diff
@@ -269,6 +295,7 @@ class Thermostat_TZE200_c88teujp(TuyaThermostat):
                 OUTPUT_CLUSTERS: [
                     Time.cluster_id,
                     Ota.cluster_id,
+                    ManufacturerThermostatCluster,
```

Comment:
Is that needed? Can't you write the attribute from the input clusters `ManufacturerThermostatCluster`?

Reply:
It is not, thanks!